easybuild/easyconfigs/n/numba/numba-0.40.0-CrayGNU-17.08-cuda9-python3.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'Bundle'
+
+name = 'numba'
+version = '0.40.0'
+
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '1.1'
+
+pyver = '%s.%s.%s' % (py_maj_ver, py_min_ver, py_rev_ver)
+pyshortver = '%s.%s' % (py_maj_ver, py_min_ver)
+
+cudamajor = '9.0'
+cudaminor = '103_3.7-6.0.4.1_2.1__g72b395b'
+cudaver = "%s.%s" % (cudamajor, cudaminor)
+
+versionsuffix = '-cuda-%s-python%s' % (cudamajor, py_maj_ver)
+
+homepage = 'http://numba.pydata.org/'
+description = """Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics,
+ Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('PyExtensions', '%s' % pyver),
+    ('cudatoolkit/%s' % cudaver, EXTERNAL_MODULE),
```

Comment:
can you please add `craype-accel-nvidia60` as dependency, as described here https://github.com/eth-cscs/production/wiki/UES-standards-(CUDA,-Python,-...)-for-EB-recipes